Modified ComplexManufacturingResult class field

ProgrammerBSeren opened the pull request:
1. added testValue field to ComplexManufacturingResult class;
2. modified result field from Double to Boolean, making it as a PASS/FAIL result;
This modification is for multi table query.

Commit:
1. added testValue field to ComplexManufacturingResult class;
2. modified result field from Double to Boolean, making it as a PASS/FAIL result;

CodeSerengetiOrg reviewed (changes requested):
ManufacturingResult class code is not modified;

Commit:
1. added testValue field to ComplexManufacturingResult class;
2. modified result field from Double to Boolean, making it as a PASS/FAIL result;

CodeSerengetiOrg reviewed (approved):
good to merge.
